### ISSUE TITLE
Support EPT point cloud as input?

### ISSUE THREAD

**Issue by joaori** [connormanning/entwine#291]
Hi,

Can EPT point clouds be used as input? The documentation page mentions that all PDAL-readable formats should be supported but [looking at the code](https://github.com/connormanning/entwine/blob/master/entwine/util/info.cpp#L361-L369) seems that files with json extension have a different parsing flow.

I've tried using a previously generated EPT scans as input for a new one without success (see below). Is there any way to do it using the current version of entwine?

```
#
# Generate an EPT point cloud
#
$ entwine build -i data/autzen.laz -o autzen-ept
1/1: data/autzen.laz
Dimensions: [
        X:int32, Y:int32, Z:int32, Intensity:uint16, ReturnNumber:uint8,
        NumberOfReturns:uint8, ScanDirectionFlag:uint8, EdgeOfFlightLine:uint8,
        Classification:uint8, ScanAngleRank:float32, UserData:uint8,
        PointSourceId:uint16, GpsTime:float64, Red:uint16, Green:uint16, Blue:uint16
]
Points: 10,653,336
Bounds: [(635577, 848882, 406), (639004, 853538, 616)]
Scale: 0.01
SRS: PROJCS["NAD_1983_HARN_Lambert_Conformal_Conic",GEOGCS["NAD83(HARN)",DATUM["NA...

Adding 0 - data/autzen.laz
Joining
00:10 - 68% - 7,266,304 - 2,615 (2,615) M/h - 9W - 0R - 163A
	Done 0
Saving
Wrote 10,653,336 points.

#
# Generate another EPT point cloud using the first as input fails...
#
$ entwine build -i autzen-ept/ept.json -o autzen-ept-2
1/1: autzen-ept/ept.json
Encountered an error: [json.exception.out_of_range.403] key 'bounds' not found
Exiting.

# Info operation also doesn't work...
$ entwine info -i autzen-ept/ept.json
Analyzing:
	Input: autzen-ept/ept.json
	Reprojection: none
	Type: shallow
	Threads: 8

1/1: autzen-ept/ept.json
	Done.

Errors:
	- autzen-ept/ept.json: Failed to fetch info: [json.exception.out_of_range.403] key 'path' not found
Encountered an error: No points found!
Exiting.
```

Thank you!

**Comment by simmers** [comment]
@joaori Hi there. Seems you're not the only one to have encountered this. Did you find a resolution yourself or did you bypass it? I'm considering an "aws sync" to my EC2, followed by a ec2-local-->s3 ept build.

**Comment by connormanning** [comment]
Closed by #308.
